src/patternfly/demos/Card/card-demo-template-gallery.hbs

```diff
@@ -0,0 +1,52 @@
+{{#> gallery gallery--modifier="pf-m-gutter"}}
+  {{#> gallery-item}}
```

Comment:
You also don't have to use a `pf-c-gallery__item` here. You can just use a `pf-c-card`.

Reply:
Oh yes nice that allows them to all be the same size according to the tallest card in the row.